### ISSUE TITLE
"Add a Repository to the Index" link not working in Contribute

### ISSUE THREAD

**Issue by rkent** [rkent/rosindex#2]
This works in the ros.org version but not rosdabbler.

**Comment by rkent** [comment]
<del>See the plugin :redcarpet_toc.rb" for the likely cause of this.</del> Nope, the line is:

```html
<a type="button" href={{ "/contribute/add_repo" | prepend: site.baseurl }} class="btn btn-success">Add a Repository to the Index</a>
```

**Comment by rkent** [comment]
This is fixed in commit b517462
